data/services.json

```diff
@@ -107,12 +107,12 @@
       "f_vsn": "1.0.0",
       "type": "authn",
       "method": "WC/RPC",
-      "uid": "https://fcw-link.lilico.app/wc",
```

Comment:
iirc this needs to match the WC side to work with fcl-wc. Just want to make sure deeplinks don't break 🙏🏼

Reply:
Sure, for now, the app supports both deeplink.